runway/hooks/staticsite/auth_at_edge/templates/shared.py

```diff
@@ -0,0 +1,363 @@
+# pylint: skip-file
+"""Shared functionality for the Auth@Edge Lambda suite.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+
+import base64
+import json
+import logging
+import re
+import time
+import traceback
+from datetime import datetime
+from random import random
+from urllib import request  # pylint: disable=no-name-in-module
+from urllib.parse import urlencode  # pylint: disable=no-name-in-module,import-error
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_config():
+    """Retrieve the configuration variables for the Auth@Edge suite.
+
+    Lambda@Edge restricts the ability to use environment variables. This configuration
+    object is generated with hard coded values via Runway.
+    """
+    # This configuration will be replaced with dynamic values
+    # via the lambda_config.py Runway hook. Please review
+    # that file for more information.
+    config = {{
+        'client_id': 'tbd',
+        'cognito_auth_domain': 'tbd',
+        'cookie_settings': {},
+        'http_headers': {},
+        'oauth_scopes': [],
+        'redirect_path_auth_refresh': 'tbd',
+        'redirect_path_sign_in': 'tbd',
+        'redirect_path_sign_out': 'tbd',
+        'user_pool_id': 'tbd',
+    }}
```

Comment:
We should make the dummy value a valid dictionary (my original suggestion was the literal `config = {'user_pool_id': 'tbd', 'http_headers': {}, 'token_issuer': 'tbd'}`, but open to alternate options like this) so the unsupported-assignment-operation warnings below can be dropped

Reply:
Sorry, wasn't getting those warnings. Will update